opm/grid/cpgrid/Entity.hpp

```diff
@@ -550,9 +519,11 @@ Dune::cpgrid::Geometry<3,3> Dune::cpgrid::Entity<codim>::geometryInFather() cons
         return Dune::cpgrid::Geometry<3,3>(center_in_father_reference_elem, volume_in_father_reference_elem,
                                            in_father_reference_elem_corners, in_father_reference_elem_corner_indices_.data());
     }
+    else {
+        OPM_THROW(std::logic_error, "Entity has no father. Entry should be -1");
+    }
```

Comment:
Mmh. Please explain what the second sentence is telling us.

Reply:
Unfortunately, it tell us implementation details that no one should care about. I'll remove that, thanks!